models/classes/accessControl/ActionAccessControl.php

```diff
@@ -147,6 +147,18 @@ private function hasAccess(array $allowedPermissions, string $controller, string
             }
         }
 
+        if (!$roleIsListed) {
+            $this->logWarning(
+                sprintf(
+                    'User roles "%s", does not have permissions "%s" form controller "%s" and action "%s',
```

Comment:
Small typo, please blame who sent this code :p 

```suggestion
                    'User roles "%s" do not have permissions "%s" form controller "%s" and action "%s',
```